src/content/docs/import/index.mdx

```diff
@@ -113,14 +88,120 @@ node table being copied into. If the node table had fewer columns than the data
+Warnings table is a connection-level table that accumulates warnings that are generated during COPY statements during the
+lifetime of a database connection. You can inspect the contents of this table, e.g., to see the warnings
+about the malformed rows, or clear this table.
+
+For example, to see the warning messages generated about the malformed line in our `user.csv` import above,
+you can run the [`SHOW_WARNINGS`](/cypher/query-clauses/call#show_warnings) function after the `COPY FROM` statement:
+
+```cypher
+CALL show_warnings() RETURN *;
+```
+
+Output:
+```
+┌──────────┬─────────────────────────────────────────────────────────────────────────────┬───────────┬─────────────┬────────────────────────┐
+│ query_id │ message                                                                     │ file_path │ line_number │ skipped_line_or_record │
+│ UINT64   │ STRING                                                                      │ STRING    │ UINT64      │ STRING                 │
+├──────────┼─────────────────────────────────────────────────────────────────────────────┼───────────┼─────────────┼────────────────────────┤
+│ 1        │ Conversion exception: Cast failed. Could not convert "2147483650" to INT32. │ user.csv  │ 2           │ Bob,2147483650         │
+└──────────┴─────────────────────────────────────────────────────────────────────────────┴───────────┴─────────────┴────────────────────────┘
+```
+
+At any point in time you can also call the [`CLEAR_WARNINGS`](/cypher/query-clauses/call#clear_warnings) function to clear the Warnings table.
+
+```cypher
+CALL clear_warnings();
+```
+
+After clearing the warnings, the Warnings table will be empty.
+
+```cypher
+CALL show_warnings() RETURN *;
+```
+
+Output:
+```
+┌──────────┬─────────┬───────────┬─────────────┬────────────────────────┐
+│ query_id │ message │ file_path │ line_number │ skipped_line_or_record │
+│ UINT64   │ STRING  │ STRING    │ UINT64      │ STRING                 │
+├──────────┼─────────┼───────────┼─────────────┼────────────────────────┤
+└──────────┴─────────┴───────────┴─────────────┴────────────────────────┘
+```
+
+By default, Kùzu stores a limited number of warnings per connection, determined by the `warning_limit` connection configuration parameter.
+You can change this configuration as follows (see the [Connection Configuration](/cypher/configuration#connection-configuration) section for more details):
+```cypher
+CALL warning_limit=1024;
+```
+### Skippable Errors By Source
+
+While the `IGNORE_ERRORS` option can be specified for all file import sources, the set of errors that can be skipped can be different. Below is a table that shows the errors that are skippable by each source.
+
+||Parsing Errors|Casting Errors|Duplicate/Null/Missing Primary-Key errors|
+|----|----|----|----|
+|CSV|&#9745;|&#9745;|&#9745;|
+|JSON|||&#9745;|
+|Numpy|||&#9745;|
+|Parquet|||&#9745;|
+|Dataframe||||
+|Subquery|||
```

Comment:
Not sure if these should be included, I included them because I thought it may be good to know that `IGNORE_ERRORS` doesn't do anything for these sources.